.github/workflows/build-and-push.yml

```diff
@@ -1,79 +1,39 @@
-name: Build and push CentOS7 images to Quay.io registry
+name: Build and push images to Quay.io registry
 on:
   push:
     branches:
       - master
+  schedule:
+      - cron: '0 1 * * 3'
 
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        version: [7.3, 7.4, 8.0]
+        dockerfile_path: ["7.3", "7.4", "8.0"]
```

Comment:
Version '8.0' is not part of the repository yet ;)

Reply:
Interesting, because it was in the build-and-push even before my proposal. should I remove it?

Reply:
@zmiklank Please remove version 8.0. It is not part of `master` branch yet ;)

Reply:
Done, sorry for the delay.